apps/src/turtle/turtle.js

```diff
@@ -86,6 +89,13 @@ var Artist = function () {
   this.level = null;
 
   this.api = new ArtistAPI();
+  apiJavascript.injectArtistAPI(this.api);
+
+  /** @type {JSInterpreter} */
+  this.JSInterpreter = null;
```

Comment:
Super small nit: lowercase `this.jsInterpreter` for the instance (constructor is capitalized).